CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ The format is based on the [KeepAChangeLog] project.
 - [#436] Fixed client.read_registration
 - [#446] Fixed provider.read_registration
 - [#449] Fixed creation of error_response on client registration
+- [#145] Succesfull token endpoint response have correct no-cache headers
```

Comment:
Successful with one 'l' only..., and should be 'responses' or 'has' (singular/plural).

Reply:
I always get this wrong :)

Spelling fixed, I will merge it after tests pass.